### ISSUE TITLE
Revert to master electron-prebuilt

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch#4]
Whenever the new electron release is out we can revert to the official version

**Comment by elisee** [comment]
As an outside observer, I'd love it if you could write rationales for the tasks you add to the bug tracker, so that I can follow along and understand not just what but why. (For instance here, it'd be neat if the ticket said why you're not using the official release atm.)

I don't mean to be a burden though, so definitely take it as nothing more than a suggestion ^_^

**Comment by fasterthanlime** [comment]
@elisee Could've sworn I linked those somewhere but here's [all the context one should need](https://github.com/mafintosh/electron-prebuilt/pull/64)

**Comment by fasterthanlime** [comment]
TL;DR
- the part of our test suite that runs in electron relies on `process.exit(code)` to actually exit with that code (indicates the number of tests failed)
- `electron-prebuilt` has a cli wrapper over the electron binary, but it always exited with 0 until my PR, no matter what the electron binary actually exited with
- the net result was that **builds on appveyor and travis were succeeding even when some tests failed**, which was definitely not good.

Also, [electron v0.33.7 is out now](https://github.com/atom/electron/releases/tag/v0.33.7), so it looks like I can address this right now!
